humanid.py

```diff
@@ -77,6 +77,16 @@ def sub(s):
         humid = separator.join([sub(word) for word in [lead, 'and', 'the', adj, self._pluralize(band)]])
         return humid if not return_hash else (hexstr, humid)
 
+    def rap_name(self, separator='_', hexstr=None, return_hash=False):
+        """ create an id in the form of a rap name: lil_blue_eyed_tube"""
+        if return_hash and hexstr is None:
+            hexstr = uuid.uuid4().hex
+        def sub(s):
+            return self._rx_non_letter.sub(separator, s)
+        (adj, noun) = self._words(hexstr, (self.adjectives, self.nouns))
+        humid = separator.join([sub(word) for word in ['lil', adj, noun]])
```

Comment:
maybe something else but just 'lil'. it could even be another adjective http://www.mess.be/inickgenwuname.php